products/cloudscheduler/api.yaml

```diff
@@ -73,6 +73,16 @@ objects:
         required: false
         input: true
         default_value: 'Etc/UTC'
+      - !ruby/object:Api::Type::String
+        name: attemptDeadline
+        description: |
+          The deadline for job attempts. If the request handler doesn't respond by this dealine,
+          the request is cancelled and the attempt is marked as failed. For example, 20s
+          Value must end with a unit: s for seconds, m for minutes, h for hours. You can combine them for example 15m20s
```

Comment:
May I ask where this documentation came from? Mostly the [Job API doc](https://cloud.google.com/scheduler/docs/reference/rest/v1/projects.locations.jobs#Job) says:

A duration in seconds with up to nine fractional digits, terminated by 's'. Example: "3.5s".